gcp_variant_transforms/libs/processed_variant.py

```diff
@@ -153,13 +153,15 @@ def __init__(self, alt_bases):
     # Note that `_info` also holds the split annotation fields. For those
     # fields, the value in the `_info` dict has a list of dicts itself.
     self._info = {}  # type: Dict[str, Any]
+    self._annotation_field_names = []  # type: Set[str]
 
   def __repr__(self):
     return ', '.join([str(self._alt_bases), str(self._info)])
 
   def __eq__(self, other):
     return (isinstance(other, AlternateBaseData) and
-            vars(self) == vars(other))
+            self._alt_bases == other._alt_bases and
```

Comment:
Are there cases where the new `_annotation_field_names` does not work anymore?

Reply:
Some unit tests didn't pass. We can also update the unit tests, but it just seems weird and unnecessary that every time we create `AlternateBaseData` in the unit tests, we have to set the the `annotation_field_names`.

Reply:
Don't worry @allieychen, it's just temporary until the recursive check implementation.